main.py

```
@@ -15,5 +15,36 @@ def main():
         else:
             print(f"{i} is odd")
 
+for i in range(5):
+        if is_even(i):
+            print(f"{i} is even")
+        else:
+            print(f"{i} is odd")
+            for i in range(5):
+        if is_even(i):
```

Ruff (0.11.9)

SyntaxError: Expected an indented block after `for` statement

---

SyntaxError: Expected an indented block after `for` statement

---

Multiple statements on one line (colon)

(E701)